governance/program/tests/program_test/mod.rs

```diff
@@ -3143,4 +3143,42 @@ impl GovernanceProgramTest {
 
         Ok(max_voter_weight_record_cookie)
     }
+
+    #[allow(dead_code)]
+    pub async fn with_complete_proposal(
+        &mut self,
+        governance_cookie: &mut GovernanceCookie,
+        proposal_cookie: &mut ProposalCookie,
+    ) -> Result<(), ProgramError> {
+        self.with_complete_proposal_using_instruction_impl(
+            governance_cookie,
+            proposal_cookie,
+            NopOverride,
+        )
+        .await
+    }
+
+    #[allow(dead_code)]
+    pub async fn with_complete_proposal_using_instruction_impl<F: Fn(&mut Instruction)>(
```

Comment:
nit: There is no need for the `_impl` suffix which is used for internal implementations